.changeset/two-planets-provide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder': major
```

Comment:
major? 😮

Reply:
oops, just wanted to create the changeset and write in the editor! :joy: